pages/profile/profilePicturesClient.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  GetObjectCommand,
+  ListObjectsV2Command,
+  S3Client,
+} from "@aws-sdk/client-s3";
+import { getSignedUrl } from "@aws-sdk/s3-request-presigner";
+
+export const fetchProfilePicture = async (
+  userId: string
+): Promise<string | null> => {
+  // fetch profile picture from aws s3 bucket
+  const s3 = new S3Client({
+    region: "us-east-1",
+    credentials: {
+      accessKeyId: process.env.NEXT_PUBLIC_AWS_ACCESS_KEY_ID,
+      secretAccessKey: process.env.NEXT_PUBLIC_AWS_SECRET_ACCESS_KEY,
+    },
+  });
+
+  const params = {
+    Bucket: "student-profile-pictures",
+    Prefix: `${userId}/`,
+  };
+  return s3
+    .send(new ListObjectsV2Command(params))
```

Comment:
to interact with an aws s3 bucket you need to understand the possible commands that you can run. the list of commands is availble at the s3 documentation page

https://docs.aws.amazon.com/AmazonS3/latest/API/API_Operations_Amazon_Simple_Storage_Service.html